.editorconfig

```diff
@@ -0,0 +1,30 @@
+# http://editorconfig.org
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+[*.{html,json}]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = false
+
+[*.{scss,js}]
+indent_style = tab
+indent_size = tab
+
+
+# Customizations for third party libraries
+
+[static/scss/{font-awesome/*.scss,_font-awesome.scss}]
+indent_style = space
+indent_size = 2
+
+[static/js/lib/**]
```

Comment:
We have bootstrap and jquery in `static/js/**`. I will probably move them to a `lib` directory.
Probably same rule should apply also to `bootstrap.min.css` and `font-awesome.min.css`.